backend/ApiGateway/ApiGateway/Configuration/Production/studio.songs.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Routes": [
+    {
+      "DownstreamPathTemplate": "/api/songs",
+      "DownstreamScheme": "http",
+      "DownstreamHostandPorts": [
+        {
+          "Host": "perflow",
+          "Port": "4203"
```

Comment:
If you're going to send requests to perflow studio, then use 'perflow-studio' as Host.
perflow:4202
perflow-studio:4203

Please replace here and in other places

Reply:
Fixed